Sources/Tracking/CustomerIO.swift

```diff
@@ -173,6 +112,38 @@ public class CustomerIO: CustomerIOInstance {
         }
     }
 
+    /**
+     Initialize the shared `instance` of `CustomerIO` with `configParams`.
+     Call this function when your app launches, before using `CustomerIO.instance`.
+     */
+    @available(iOSApplicationExtension, unavailable)
+    public static func initialize(
+        siteId: String,
+        apiKey: String,
+        region: Region,
+        params: [String: Any],
+        configure configureHandler: ((inout SdkConfig) -> Void)?
+    ) {
```

Comment:
Is there a different solution that we can introduce then adding another set of `initialize` functions? I want to avoid us having 3 or 4 different `initialize` functions if we can help it. Already needing 2 requires maintenance. 

# Idea 

What about adding a function to `SdkConfig` object: `SdkConfig.modify([String: String])` that's job is to modify the object? It would be similar to `SdkConfig.create()` that you added, but it modifies the object instead of creating a new one? 

That way, we don't need to add a new `CustomerIO.initialize()` function, but SDK wrappers can use the existing `initialize` functions like this:

```
CustomerIO.initialize(...) { config in 
  config.modify(["foo": "bar"])
}
```

Reply:
I am not fully onboard with the idea, of the `modify` method. 

I do understand that we don't want to increase the number of `initialize` methods, so what I am going to do is combine them with a default empty value, since they both do almost the same thing.

This way, we don't create new methods, previous method doesn't break either.

Reply:
Did you make a change yet? I was expecting to no longer see a new `initialize` function in the `CustomerIO` class in this pull request as I currently do. 

Did you have an idea in mind for how we could avoid adding a new `initialize` function?